src/Arcus.Security.Core/Storage/Source/ConfigurationSecretProvider.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Threading.Tasks;
```

Comment:
Let's move this to `Arcus.Security.Core/Providers`

Reply:
Ok, we're leaving the "topic" stuff behind then 😄 .

Reply:
For concrete implementations, I can agree.

Reply:
Done